qemu.go

```diff
@@ -985,6 +996,110 @@ func (q *qemu) hotplugRemoveDevice(devInfo interface{}, devType deviceType) erro
 	return q.pod.storage.storeHypervisorState(q.pod.id, q.state)
 }
 
+func (q *qemu) hotplugCPU(amount uint32, op operation) error {
```

Comment:
I was going to say that this function is quite big so might be better split up:

```
if op == addDevice {
    q.addCPU(...)
} else {
    q.removeCPU(...)
}
```
... but then I saw `hotplugAddCPUs()` and `hotplugRemoveCPUs()`, but those two aren't used (yet?)

I assume this is an early branch but we should add some unit tests (even if they only test failure scenarios).

Reply:
yep, I still need to fix other things